docs/reference/commandline/ps.md

```diff
@@ -24,6 +24,7 @@ weight=1
       -q, --quiet=false     Only display numeric IDs
       -s, --size=false      Display total file sizes
       --since=""            Show created since Id or Name, include non-running
+      -F, --format=[]       Pretty-print containers using a Go template
```

Comment:
Still +1 on removing it for now. We can add it later if users request it to be added.